include/picongpu/particles/atomicPhysics2/rateCalculation/BoundBoundTransitionRates.hpp

```diff
@@ -122,8 +121,8 @@ namespace picongpu::particles::atomicPhysics2::rateCalculation
                 g = A * math::log(U) + B + C / (U + a) + D / ((U + a) * (U + a)); // unitless
```

Comment:
If you move `math::log(U)` before the loop the code will be faster `auto const logU = math::log(U);`

Reply:
**Attention** possible division by zero in case where `(U + a) == 0` This should be handled!

Reply:
fixed